### DISCUSSION TITLE
Quelqu'un pourrais me dire d'où viens le problème ?

### DISCUSSION THREAD

**Discussion by NH-Rayvers** [community/community#64186] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

nom = input ( "Comment vous appelez vous ?")
print ("Bonjour " +nom)
Nourriture = input("Quel est ton plat préférer ?" )
Naissance = input ("Quelle est ton année de naissance ?" )
age = 2023 - int(Naissance)
print(nom,  'aime le', Nourriture ,age, 'an' )

**Comment by queenofcorgis** [comment]
Hi @NH-Rayvers , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
